lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -653,6 +653,10 @@ namespace Js
                 DynamicType::New(scriptContext, TypeIds_Object, objectPrototype, nullptr, typeHandler, true, true);
         }
 
+        SimplePathTypeHandler * typeHandler = SimplePathTypeHandler::New(scriptContext, this->GetRootPath(), 0, 0, sizeof(DynamicObject), true, true);
+        typeHandler->SetIsInlineSlotCapacityLocked();
```

Comment:
What does this do, and why is it necessary?  (Not a problem; I'm just asking for my own information.)

Reply:
I more or less blindly copied this from the normal ObjectType creation code. It prevents inline slot capacity from being reduced, and I think that actually doesn't do anything here because inline slot capacity is 0. Removed.